app/views/application/exception.js.erb

```diff
@@ -0,0 +1,6 @@
+var message = '<%= j(t(:internal_server_error) + ' ' + @message) %>';
+if ($('#modal').hasClass('hide')) {
+  $('.content.container').prepend('<div class="alert alert-danger" role="alert">' + message + '</div>');
+} else {
+  $('#modal .modal-body').html(message);
+}
```

Comment:
Im `master` wurde die Datei bereits auf coffee transformiert, vermutlich hat sich das mit diesen Ergänzungen überschnitten. Bitte nach einem Rebase auf den aktuellen `master ` die Änderungen in CoffeeScript ergänzen.